proto/cosmos/gov/v1/tx.proto

```diff
@@ -125,3 +130,19 @@ message MsgUpdateParams {
 //
 // Since: cosmos-sdk 0.47
 message MsgUpdateParamsResponse {}
+
+// MsgCancelProposal is the Msg/CancelProposal request type.
+//
+// Since: cosmos-sdk 0.47 
+message MsgCancelProposal{
+  option (cosmos.msg.v1.signer) = "proposer";
+  
+  uint64  proposal_id = 1 [(gogoproto.jsontag) = "proposal_id"];
+  string  proposer    = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// MsgCancelProposalResponse defines the response structure for executing a
+// MsgCancelProposal message.
+//
+// Since: cosmos-sdk 0.47
+message MsgCancelProposalResponse{}
```

Comment:
Perhaps we add a field stating the time and/or height at which it was cancelled?

Reply:
We need to start adding relevant info (that isn't part of the initial message) to response objects.

Reply:
@anilCSE  @alexanderbez  
```proto
message MsgCancelProposalResponse{
   uint64 proposal_id  = 1;
   google.protobuf.Timestamp         canceled_time = 2 [(gogoproto.stdtime) = true];
   uint64 height = 3;

}
```
this fields are enough for response of cancel proposal ?